app/services/election_service.rb

```diff
@@ -0,0 +1,20 @@
+module ElectionService
+  def self.create_candidate(candidate, user)
+    candidate.user == user
+    candidate.validate!
+    if !candidate.election.current_posts.exists?(candidate.post.id)
+      candidate.errors.add(:post, I18n.t("elections.not_allowed_post"))
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.